frontend/src/views/TopLevelSearchView/TopLevelSearchView.spec.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Really basic unit test to demo how to write unit tests.
We should write more :)